src/splitInteger.test.js

```diff
@@ -4,16 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-  expect(splitInteger(17, 4))
-    .toHaveLength(4);
+  expect(splitInteger(10, 2)).toEqual([5, 5]);
```

Comment:
The test description does not match the test implementation. The description talks about splitting a number into equal parts if it's divisible, but the test checks for the specific case of 10 divided into 2 parts.

Reply:
> 

Sorry, already fixed my mistakes